src/samples/java/com/box/sdk/samples/appusers/helpers/BoxHelper.java

```diff
@@ -0,0 +1,234 @@
+package com.box.sdk.samples.appusers.helpers;
+
+
+import java.io.InputStream;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpSession;
+
+import com.box.sdk.*;
+
+/**
+ * BoxHelper.  A Helper class for managing Box App User's/
+ * Stores and retwieves names and is in the current HTTP Session Object
+ * Ceates ne App users or looks up existin app users by name within the enterprise
+ * associated with this applications.
+ */
+public final class BoxHelper {
+
+    static final String ENTERPRISE_ID = ConfigHelper.properties().getProperty("boxEnterpriseId");
+    static final String CLIENT_ID = ConfigHelper.properties().getProperty("boxClientId");
+    static final String CLIENT_SECRET = ConfigHelper.properties().getProperty("boxClientSecret");
+    static final String PRIVATE_KEY_FILE = ConfigHelper.properties().getProperty("boxPrivateKeyFile");
+    static final String PRIVATE_KEY_PASSWORD = ConfigHelper.properties().getProperty("boxPrivateKeyPassword");
+    static final String PUBLIC_KEY_ID = ConfigHelper.properties().getProperty("boxPublicKeyId");
+
+    static final String BOX_USER_ID_KEY = "box_id";
+    static final String BOX_USER_NAME = "name";
+
+    static final int MAX_CACHE_ENTRIES = 100;
+
+    private static IAccessTokenCache accessTokenCache;
+    private static JWTEncryptionPreferences jwtEncryptionPreferences;
+
+    /**
+     * Private constructor since this is a utility class.
+     */
+    private BoxHelper() {
+
+    }
+
+    static {
+        String privateKey;
+        try {
+            privateKey = new String(Files.readAllBytes(Paths.get(PRIVATE_KEY_FILE)));
+        } catch (Exception ex) {
+            throw new BoxAPIException("Unable to read private key file: " + PRIVATE_KEY_FILE);
+        }
+
+        jwtEncryptionPreferences = new JWTEncryptionPreferences();
+        jwtEncryptionPreferences.setPublicKeyID(PUBLIC_KEY_ID);
+        jwtEncryptionPreferences.setPrivateKey(privateKey);
+        jwtEncryptionPreferences.setPrivateKeyPassword(PRIVATE_KEY_PASSWORD);
+        jwtEncryptionPreferences.setEncryptionAlgorithm(EncryptionAlgorithm.RSA_SHA_256);
+
+        accessTokenCache = new InMemoryLRUAccessTokenCache(MAX_CACHE_ENTRIES);
+    }
+
+    /**
+     * Uses the statically configured parameters to create Developer Edition connection.
+     *
+     * @return A new Box Developer Edition API connection with enterprise token leveraging an access token cache.
+     */
+    public static BoxDeveloperEditionAPIConnection adminClient() {
```

Comment:
It is here declared as a static as the Java standard way of creating a singleton